Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -646,558 +587,373 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        /// <summary>
-        /// Tries to identify the string and return an object of that class.
-        /// </summary>
-        /// <param name="addr">String to parse.</param>
-        /// <param name="result">IPObject to return.</param>
-        /// <returns><c>true</c> if the value parsed successfully, <c>false</c> otherwise.</returns>
-        private static bool TryParse(string addr, out IPObject result)
+        /// <inheritdoc/>
+        public bool TryParseInterface(string intf, [NotNullWhen(true)] out IReadOnlyList<IPData>? result)
         {
-            if (!string.IsNullOrEmpty(addr))
+            var resultList = new List<IPData>();
+            if (string.IsNullOrEmpty(intf) || _interfaces is null)
```

Comment:
If neither IPv4 nor IPv6 is enabled, will this method get called? If so, you should add it as an early exit here

Reply:
The function matches against `_interfaces` which already excludes all adapters/interfaces not adhering to the setting.